auacm/app/static/js/controllers/ScoreboardController.js

```diff
@@ -107,22 +107,22 @@ app.controller('ScoreboardController', ['$scope', '$http', '$routeParams',
         .then(function(response) {
             $scope.competition = response.data.data.competition;
             $scope.compProblems = response.data.data.compProblems;
-            $scope.problemNames = response.data.data.names;
+            $scope.problemNames = Object.keys(compProblems).sort();
```

Comment:
This gives an error. Instead of `compProblems` it should be `$scope.compProblems`